Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/triangulate_hole.h

```diff
@@ -99,13 +102,19 @@ namespace Polygon_mesh_processing {
 #endif
 
     CGAL_precondition(face(border_halfedge, pmesh) == boost::graph_traits<PolygonMesh>::null_face());
+    bool use_cdt =
+    #ifdef CGAL_HOLE_FILLING_DO_NOT_USE_CDT2
+        false;
+#else
+        choose_parameter(get_parameter(np, internal_np::use_2d_constrained_delaunay_triangulation), true);
```

Comment:
I don't know if the default should be `true`. opinions?

Reply:
I'd say it depends of the most usual cases. IMHO, if most of the time, a hole is not planar, it could be debated, but if most holes are small and planar, it should be `true` as it will be more stable.

Reply:
In my experience, most of the holes can be triangulated with CDT. Even when I was creating a test where it is not the case, I had to take quite a specific part of the mesh. I am not an expert in holes filling, so maybe I miss something, but I would use `true`.

Reply:
To be on the safe side, I'll put it to `false` by default and while we test it on real cases, we can reevaluate the decision and switch to `true` by default.

Reply:
Looks like the default is still to true while we decided it should be false.